metrics/custom.go

```diff
@@ -8,6 +8,26 @@ const (
 	promNS = "gopherciser"
 )
 
+// ApiCallDuration histogram for API call duration
+var ApiCallDuration = prometheus.NewHistogramVec(
+	prometheus.HistogramOpts{
+		Name:    "api_request_duration_seconds",
+		Help:    "A histogram of HTTP request durations.",
+		Buckets: []float64{0.005, 0.01, 0.025, 0.05, 0.1, 0.15, 0.25, 0.5, 1., 2.0, 2.5, 5., 10.},
+	},
+	[]string{"path", "method", "status_code"},
+)
+
+// ApiCallDurationQuantile summary for API call duration
+var ApiCallDurationQuantile = prometheus.NewSummaryVec(
+	prometheus.SummaryOpts{
+		Name:       "api_request_duration_quantiles_seconds",
+		Help:       "A summary of HTTP request durations",
+		Objectives: map[float64]float64{0.1: 0.1, 0.5: 0.05, 0.95: 0.01, 0.99: 0.001, 0.999: 0.0001},
```

Comment:
Is this correct? Copy-pasted from app-eval, although it looks a bit strange to me.

Reply:
Looks superweird, but seems consistent with this
https://github.com/prometheus/client_golang/blob/babeb356a51b1277535260e0122ad8c46240ab96/prometheus/examples_test.go#L311